OpenSim/Simulation/Model/ModelComponent.cpp

```diff
@@ -76,7 +79,16 @@ void ModelComponent::extendConnect(Component &root)
         connectToModel(*model);
 }
 
-
+void ModelComponent::extendFinalizeFromProperties()
+{
+    Super::extendFinalizeFromProperties();
+    int geomSize = getProperty_GeometrySet().size();
+    if (geomSize > 0){
+        for (int i = 0; i < geomSize; ++i){
+            addComponent(&upd_GeometrySet(i));
```

Comment:
@aseth1, I'm confused. I'm sitting in the debugger with a breakpoint in this routine and can see that it is called from the constructor `Model::Model()`. [Here](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Simulation/Model/Model.cpp#L98) is the code (from the master branch). Please clarify.